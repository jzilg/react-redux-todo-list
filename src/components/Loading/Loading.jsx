import React from 'react'
import spinner from './spinner.svg'
import style from './loading.scss'

const Loading = () => (
    <div className={style.loading}>
        <img src={spinner} alt="Loading..." />
    </div>
)

export default Loading
