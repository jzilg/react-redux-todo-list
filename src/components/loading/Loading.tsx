import React, { ReactElement } from 'react'
import spinner from './spinner.svg'
import style from './loading.scss'

const Loading = (): ReactElement<{}> => (
    <div className={style.loading}>
        <img src={spinner} alt="Loading..." />
    </div>
)

export default Loading
