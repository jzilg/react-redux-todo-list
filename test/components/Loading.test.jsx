import React from 'react'
import { shallow } from 'enzyme'
import { shallowToJson } from 'enzyme-to-json'
import Loading from '../../src/components/loading'

describe('Loading', () => {
    it('should render correctly', () => {
        const component = shallow(<Loading />)
        const tree = shallowToJson(component)
        expect(tree).toMatchSnapshot()
    })
})
