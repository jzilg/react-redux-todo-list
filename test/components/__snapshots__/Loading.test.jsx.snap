// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading should render correcty 1`] = `
<div
  styleName="loading"
>
  <img
    alt="Loading..."
    src=""
  />
</div>
`;
